vendor/fasttext/curlstream.h

```diff
@@ -0,0 +1,39 @@
+#include <iostream>
+#include <streambuf>
+#include <curl/curl.h>
+#include <string.h>
+
+class CurlStreambuff : public std::streambuf {
+public:
+  explicit CurlStreambuff(const std::string& url);
+  ~CurlStreambuff();
+
+  // Copy and move prohibited
+  CurlStreambuff(const CurlStreambuff&) = delete;
+  CurlStreambuff& operator = (const CurlStreambuff&) = delete;
+  CurlStreambuff(CurlStreambuff && src) = delete;
+  CurlStreambuff& operator = (CurlStreambuff && rhs) = delete;
+
+protected:
+  std::streamsize xsgetn(char *s, std::streamsize n) override;
+  int underflow() override;
+  int uflow() override;
+
+private:
+  static int writer_callback(char *data, size_t sz, size_t nmemb, void* ptr);
```

Comment:
Short var names here.

Reply:
I copied these from the libcurl examples, but yeah, I can change them.

Reply:
Thank you!